### ISSUE TITLE
Dbeaver's Snapcraft package is missing font files under WSL

### ISSUE THREAD

**Issue by linghengqian** [dbeaver/dbeaver#35028]
### Description

- Dbeaver's Snapcraft package is missing font files under WSL. This was confirmed at #22777 .
- The symptoms of this issue are described below. Considering this is a snapcraft package for dbeaver, I thought it would be reasonable to build more font files into the snapcraft package.
```shell
sudo apt update && sudo apt upgrade -y
sudo snap install dbeaver-ce
snap run dbeaver-ce

# Close dbeaver first, then try to install some system fonts
sudo apt-get install fonts-noto-cjk fonts-noto-color-emoji -y
snap run dbeaver-ce
```

### DBeaver Version

CE 24.1.3

### Operating System

Ubuntu 22.04.4 WSL

### Database and driver

- Unnecessary.

### Steps to reproduce

- Use Ubuntu WSL 22.04.4 from https://www.microsoft.com/store/productId/9PN20MSR04DW as the verified distribution.
```bash
sudo apt update && sudo apt upgrade -y
sudo snap install dbeaver-ce
snap run dbeaver-ce
```
- Open Dbeaver settings, switch to Simplified Chinese, and restart.
- ![image](https://github.com/user-attachments/assets/b877ba75-bd53-43d5-a3f8-fe1486104cab)
- ![image](https://github.com/user-attachments/assets/165e4a99-6f6e-4b9a-8686-82c10d1dd861)
- ![image](https://github.com/user-attachments/assets/efad279d-e32d-47f9-a9bd-06391eeb9e51)
- ![image](https://github.com/user-attachments/assets/ebcb23f8-2140-4fce-bf9e-8e9ad2f4ba26)
- At this point, the entire dbeaver displays various garbled texts due to the lack of corresponding fonts.
- ![image](https://github.com/user-attachments/assets/6e858d54-1d5b-4688-af21-d6cc6c701a0c)
- ![image](https://github.com/user-attachments/assets/598f3a65-2bde-45b5-ae54-81af0ef1f7f7)
- It seems that dbeaver has no documentation on what fonts to use. I tested further using `fonts-noto-cjk` and `fonts-noto-color-emoji`.
```bash
sudo apt-get install fonts-noto-cjk fonts-noto-color-emoji -y
snap run dbeaver-ce
```
- After installing a certain font, most of the garbled text began to return to normal, but some fonts still could not be displayed.
- ![image](https://github.com/user-attachments/assets/e58ba489-2177-4997-839b-482f3ce82374)
- Considering this is a snapcraft package for dbeaver, I thought it would be reasonable to build more font files into the snapcraft package.

### Additional context

- Unnecessary.

**Comment by linghengqian** [comment]
- I made another attempt. I reset the WSL distribution and then installed the fonts before installing dbeaver.
```shell
sudo apt update && sudo apt upgrade -y
sudo apt-get install fonts-noto-cjk fonts-noto-color-emoji -y
sudo snap install dbeaver-ce
snap run dbeaver-ce
```
- Now all fonts display properly.
- ![image](https://github.com/user-attachments/assets/4bf4e24b-bdb5-4e92-8a65-6706c8e0f1cf)


> Regarding the fonts, you can check, which font is used and change it in preferences. As far as I see, it's a Banner font and/or Header font.

- @E1izabeth I checked the font settings at this time. Banner font and Header font are both `Sans Bold`.
- ![image](https://github.com/user-attachments/assets/47651cea-0f48-46e3-bf32-657aeff37ecb)
- ![image](https://github.com/user-attachments/assets/450c23f9-e3b4-4b9c-9ad5-9ec83bfdc03b)
- ![image](https://github.com/user-attachments/assets/448d77b1-f74b-40c4-b97a-8ee67cc1d8a0)
- `Sans Bold` is actually installed by `sudo apt-get install fonts-noto-cjk fonts-noto-color-emoji -y`. So my question is, why doesn't Snapcraft package come with its own font?
